src/features/profile/ui/user-comment-list.tsx

```
@@ -0,0 +1,111 @@
+              <ThumbsUp className="h-4 w-4" />
+              <span>{comment.reactionCount}</span>
+            </div>
+          )}
+        </div>
+
+        <div className="relative mb-3">
+          <div
+            className={`prose prose-sm dark:prose-invert max-w-none overflow-hidden transition-all duration-300 ${
+              isExpanded ? '' : 'max-h-[150px]'
+            }`}
+            dangerouslySetInnerHTML={{ __html: comment.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)